pkg/templates/charts/toggle/multicloud-operators-subscription/templates/argocd-pull-integration-controller-manager.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: argocd-pull-integration
+    app.kubernetes.io/instance: controller-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: deployment
+    app.kubernetes.io/part-of: argocd-pull-integration
+    control-plane: controller-manager
+  name: argocd-pull-integration-controller-manager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      control-plane: controller-manager
+  template:
+    metadata:
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+      creationTimestamp: null
+      labels:
+        control-plane: controller-manager
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
+                - arm64
+                - ppc64le
+                - s390x
+              - key: kubernetes.io/os
+                operator: In
+                values:
+                - linux
+      containers:
+      - args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=0
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
```

Comment:
I think there will be some broader discussion about this PR, but I was just glancing at this and see this hardcoded image ref to something coming out of Google container registry.  This will not be acceptable for the productized ACM.  All images must be productized ones served out of registry.redhat.io.  Besides that, the image will have to appear in the image manifest, be referenced by an image override, etc.

Reply:
Thanks for the review Joe. We will remove this additional deployment.
This came from the kubebuilder project bootstrap.